assets/i18n/en.json

```diff
@@ -621,15 +621,17 @@
     "I18N_SAVE_PROGRESS": "Log in or sign up to save your progress and play through the next lesson.",
     "I18N_SHARE_LESSON": "Share this lesson",
     "I18N_SHOW_SOLUTION_BUTTON": "Show Solution",
-    "I18N_SIDEBAR_ABOUT_LINK": "About Oppia",
+    "I18N_SIDEBAR_ABOUT_LINK": "About US",
```

Comment:
Also for all the new keys created we will have to add a description in qqq.json. 
Can we have it as "About Us" and use 'capitalization' pipe in frontend?

Reply:
done!